pyccel/ast/builtins.py

```diff
@@ -457,3 +516,18 @@ def python_builtin_datatype(name):
 
     return None
 
+builtin_functions_dict = {
+    'abs'      : PythonAbs,  # TODO: create a built-in Abs
+    'range'    : Range,
+    'zip'      : Zip,
+    'enumerate': Enumerate,
```

Comment:
I assume you are leaving the renaming for another PR? In this case why is this PR marked as closing issue #258 ?

Reply:
I think we don't need to rename the python syntax things.

Reply:
I think it's not as serious as there are no name collisions for these objects. But the aim is to have a new naming convention, so there is no reason for this to not include these classes

Reply:
ok. I will take care of that.